src/service/viewport/viewport-binding-ios-embed-wrapper.js

```diff
@@ -44,14 +45,15 @@ export class ViewportBindingIosEmbedWrapper_ {
     /** @const {!Window} */
     this.win = win;
 
-    const topClasses = this.win.document.documentElement.className;
-    this.win.document.documentElement.className = '';
-    this.win.document.documentElement.classList.add('i-amphtml-ios-embed');
+    const documentElement = this.win.document.documentElement;
+    const topClasses = documentElement.className;
+    documentElement.className = 'i-amphtml-ios-embed';
```

Comment:
Nice clean up. 

I'm curious why className was set to an empty value and the classList was added to afterward in the previous code. Is there a subtle behaviour specific to iOS?

Reply:
Looks like an oversight from https://github.com/ampproject/amphtml/pull/6276/files#diff-6eedd5ca45df1b5aee7d616b017d360aR1419.